subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,93 @@
+# Procedure - Debriefing Session
+
+This document intends to describe how debriefing sessions
+are organized so that it continue even after members go away.
```

Comment:
either "continue**s** even after **the current facilitators**" or something like "it is sustainable"